core/src/main/java/org/apache/iceberg/IncrementalFileCleanup.java

```diff
@@ -260,10 +260,18 @@ public void cleanFiles(TableMetadata beforeExpiration, TableMetadata afterExpira
         findFilesToDelete(manifestsToScan, manifestsToRevert, validIds, afterExpiration);
 
     deleteFiles(filesToDelete, "data");
-    LOG.warn("Manifests to delete: {}", Joiner.on(", ").join(manifestsToDelete));
-    LOG.warn("Manifests Lists to delete: {}", Joiner.on(", ").join(manifestListsToDelete));
+    LOG.debug("Manifests to delete: {}", Joiner.on(", ").join(manifestsToDelete));
+    LOG.debug("Manifests Lists to delete: {}", Joiner.on(", ").join(manifestListsToDelete));
     deleteFiles(manifestsToDelete, "manifest");
     deleteFiles(manifestListsToDelete, "manifest list");
+
+    if (!beforeExpiration.statisticsFiles().isEmpty()) {
+      Set<String> expiredStatisticsFilesLocations =
+          expiredStatisticsFilesLocations(beforeExpiration, afterExpiration);
+      LOG.debug(
+          "Statistics files to delete: {}", Joiner.on(", ").join(expiredStatisticsFilesLocations));
```

Comment:
What is the value of this debug message?

Reply:
Yes. It is only present in incremental flow (not reachable flow). I just followed the same pattern which is present for manifest, manifest list files. Shall I remove them too?